requirements.txt

```diff
@@ -1,5 +1,4 @@
-fasm
-futures
+pyfasm
```

Comment:
This is incorrect, as fasm is referring to https://github.com/SymbiFlow/fasm not to https://github.com/srounet/pyfasm.  Maybe updated it something like https://stackoverflow.com/questions/16584552/how-to-state-in-requirements-txt-a-direct-github-source ?